cmd/build/helmify/static/templates/probe-webhook-post-install.yaml

```diff
@@ -47,5 +47,6 @@ spec:
       nodeSelector:
         {{- toYaml .nodeSelector | nindent 8 }}
       {{- end }}
+  backoffLimit: 0
```

Comment:
we updated `backOffLimit` to 3 as part of https://github.com/open-policy-agent/gatekeeper/pull/2710, should we use the same value here too?

Reply:
3 makes sense, just in case the initial Job fails. That would also make the default timeout 60 seconds * 3 backOffs = 180 seconds which is ample time for Gatekeeper to start up. Made the change!